templates/homepage.hamlet

```diff
@@ -1,30 +1,53 @@
-<div .col .col-main>
-  <h2>
+<div .col>
+  <h1>
     <strong>Pursuit
-    is the home of PureScript documentation.
-  <p>
-    Pursuit hosts generated API documentation for PureScript packages.
+    is the home of PureScript documentation
+
+<div .col.col--main>
   <p>
-    If you are a package author looking for information on how to publish your
-    packages, see the <a href=@{HelpR}>Help page</a>.
+    Pursuit hosts API documentation for <a href="http://www.purescript.org/">PureScript</a> packages. It lets you search by package, module, and function names, as well as approximate type signatures.
+
   <p>
-    Pursuit is free and open-source software, and the code is hosted on GitHub.
-    Contributions are welcome: <a href="https://github.com/purescript/pursuit">https://github.com/purescript/pursuit</a>.
+    To get started, have a look around the <a href="/packages/purescript-prelude">Prelude</a> – PureScript’s standard library – or try one of these example searches:
+
+  <ul .ul--search>
+    <li><a href="/search?q=map">map</a></li>
+    <li><a href="/search?q=%28a+->+b%29+->+f+a+->+f+b">(a -> b) -> f a -> f b</a></li>
+    <li><a href="/search?q=Data.List">Data.List</a></li>
+
+<div .col.col--aside>
+  <dl .grouped-list style="margin-top: 0.6em">
+    <dt .grouped-list__title>Publish
+    <dd .grouped-list__item>
+      If you are a package author looking for information on how to publish your
+      packages, see the <a href=@{HelpR}>Help page</a>.
 
+  <dl .grouped-list>
+    <dt .grouped-list__title>Contribute
+    <dd .grouped-list__item>
+      Pursuit is free and open-source software, and the code is hosted on GitHub.
+      Contributions are welcome: <a href="https://github.com/purescript/pursuit">https://github.com/purescript/pursuit</a>.
+
+<div .clear-floats>
   $if null pkgNames
-    <h2>No packages uploaded yet.
+    <h2>Package index
+    <p>No packages uploaded yet.
   $else
-    <h2>Latest Uploads
-    <ul>
-      $forall (pkgName, version) <- latest
-        <li>
-          <a href=@{PackageVersionR (PathPackageName pkgName) (PathVersion version)}>
-            #{runPackageName pkgName} - #{showVersion version}
-    <h2>Packages:
-      $forall pkgs <- pkgNamesByLetter
-        $forall letter <- headMay pkgs >>= firstLetter
-          <h3>#{letter}
+    <div .multi-col .clearfix>
+      <div .multi-col__col>
+        <h3>Latest packages
```

Comment:
Also to answer your other question, yes, I think ideally "uploader" is the person who uploaded it, and "publisher" is the person who created the git tag (although I don't think it's all that easy for us to work out who created the git tag right now).